### ISSUE TITLE
Can't enter on "Users" Page

### ISSUE THREAD

**Issue by brylaaa** [owncloud/core#28005]
Hi. I have trouble getting into the users panel. When I click on the icon, the page is constantly loading but not open. When I want to open another page later, the page does not respond. Only after web browser restart I can eneter on my owncloud web interface again.

**Comment by kdslkdsaldsal** [comment]
@brylaaa Please fill out the issue template shown when creating a new issue here. Currently even the basic info about your environment is missing.

**Comment by kdslkdsaldsal** [comment]
Closing as the info asked wasn't provided after more then one month.
